src/app/widgets/chart/chart.settings.tpl.html

```diff
@@ -33,8 +35,13 @@ <h3 translate-keep-content translate="widget.chart.settings.header">Chart Settin
                     </div>
                     <div class="form-group" ng-class="{error: _form.name.$error && _form.submitted}" ng-if="form.charttype=='interactive'">
                         <label class="control-label col-lg-3 col-md-3" translate-keep-content translate="widget.chart.general.service">Persistence service</label>
-                        <div class="col-lg-9 col-md-9">
-                            <input name="name" type="text" ng-model="form.service" class="form-control" placeholder="{{'rrd4j, mysql, influxdb... Leave blank for default' | translation:'widget.chart.general.service.hint'}}" />
+                        <div class="col-lg-9 col-md-9">                            
+                            <!--input name="name" type="text" ng-model="form.service" class="form-control" placeholder="{{'rrd4j, mysql, influxdb... Leave blank for default' | translation:'widget.chart.general.service.hint'}}" /-->
+                            <select ng-model="form.service" class="form-control">

```

Comment:
maybe we should add also a textbox to enter another instance (if it is not in the list)

Reply:
means also, atm. this setting doesn't do anything, because getHistory uses default instance?

Reply:
for now all possible versions are in. I aggree, that maybe a textbox or list of available adapter would be beneficial. right now it is the first minimum viable solution